tests/emukit/quadrature/test_vanilla_bq.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+
+import numpy as np
+import mock
+from numpy.testing import assert_array_equal
+
+from emukit.quadrature.interfaces.base_gp import IBaseGaussianProcess
+from emukit.quadrature.methods.vanilla_bq import VanillaBayesianQuadrature
+
+
+def test_vanilla_bq_transformations():
+    Y = np.random.rand(5, 1)
+
+    mock_gp = mock.create_autospec(IBaseGaussianProcess)
+    method = VanillaBayesianQuadrature(base_gp=mock_gp)
+
+    assert_array_equal(method.inverse_transform(Y), Y)
```

Comment:
So identical transform? Just curious. How would that look for a different quadrature method?

Reply:
for square transform you would have e.g.

```
Y = np.array([[4, 9, 16]])
Y_inv_transform = np.array([[2, 3, 4]])
Y_transform = np.array([[16, 81, 256]])

assert_array_equal(method.inverse_transform(Y), Y_inv_transform)
assert_array_equal(method.transform(Y), Y_transform)
assert_array_equal(method.inverse_transform(method.transform(Y)), Y)
```

but maybe there is a better way to test this?